datadog_checks_dev/datadog_checks/dev/tooling/cli.py

```diff
@@ -57,6 +58,9 @@ def ddev(ctx, core, extras, agent, here, color, quiet):
     set_root(root)
     set_color(config['color'])
 
+    if debug:
```

Comment:
Should `--quiet` keep priority? (Not sure, I could roll either way)

```suggestion
    if debug and not quiet:
```